hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -548,5 +549,18 @@ public String createSnapshot(String volumeName,
     return proxy.createSnapshot(volumeName, bucketName, snapshotName);
   }
 
-
+  /**
+   * List snapshots in a volume/bucket, filtered by a name prefix.
+   * @param volumeName volume name
+   * @param bucketName bucket name
+   * @param startKey   the start snapshot name
```

Comment:
Whats the use case for using the startKey param ?

Reply:
The API could be made rich or could be simple. The Base API could be simple here, returning all snapshots for a given bucket. You can add richness of parameters on top of that e.g. filtering capability based on name-pattern, name-range, creation-time range etc.

Reply:
OK, I'll make this pr simple first.